register.go

```diff
@@ -1,3 +1,4 @@
+//nolint:revive,stylecheck
```

Comment:
What is the error?

Reply:
It requires that the package is documented. But the register code is essentially plumbing, and I didn't judge it necessary to document it. Let me know if you disagree 👍🏻

Reply:
I think it's difficult to remember, so we should add a comment for it, but at that point, we can directly comment with an obvious sentence and remove the nolint.